### PR TITLE
[language] add Move release notes for version 1.4

### PR DESCRIPTION
**Pull request by bob-wilson**
I reviewed the Move changes in the 1.4 release and produced this set of release notes.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Viewed formatted Markdown

### CODE REVIEW

File: language/RELEASES.md [hunk 1]
```diff
@@ -1,3 +1,91 @@
...
+and refactored the Move CLI so that `df-cli` (or other clients) can
+extend it with new subcommands
+([#8764](https://github.com/diem/diem/pull/8764)).
+
+## Miscellaneous
+
+* Updated the Move language book to use abilities
+([#8582](https://github.com/diem/diem/pull/8582)).
+The documentation still needs more updates to catch up with the latest features.
+* Finished the process of removing Cargo dependencies on Diem crates,
```
Comment (sausagee) [inline language/RELEASES.md:73]: IMO this refactoring is note worthy. Do we want to move this to the highlight section above?

**Review by sausagee — APPROVED** [review]
I went through the release note and it is consistent with Diem Core's v1.4 changelog.  There is a suggestion on highlighting the dependency edges clean up.  Bob, I will defer the nit to you.

File: language/RELEASES.md (hunk 1 above)
Comment (bob-wilson) [reply language/RELEASES.md:73]: I'd personally wait until we have a stronger position for someone to actually use the Move VM apart from Diem. Getting rid of the Cargo dependencies was indeed a significant milestone, but we still have other implicit dependencies on Diem. There's still quite a bit of work to do.

**Comment by bob-wilson** [comment]
I'll go ahead and land this but if anyone has further comments or suggestions, please speak up. I'm happy to edit this after it lands.

**Pull request merged**